apps/src/componentLibrary/tabs/tabs.module.scss

```diff
@@ -149,7 +278,7 @@
         @include label-one;
         margin: 0;
         padding: 0 0.75rem;
-        gap: 0.4375rem;
+        gap: 0.375rem;
         height: 2rem;
 
         &:has(i:first-child) {
```

Comment:
Wondering if we could maybe vertically center the tab text a bit more...Could update `span` margin top to a slightly bigger value (e.g. line 167) to `margin: 3px 0 0;`.